packages/sw-build-webpack-plugin/index.js

```diff
@@ -0,0 +1,65 @@
+const swBuild = require('sw-build');
+const path = require('path');
+const SERVICE_WORKER_NAME = 'sw.js';
```

Comment:
@addyosmani @jeffposnick should we default the dest to `<rootDirectory>/sw.js` in sw-build as well?